test/battle/weather/sandstorm.c

```diff
@@ -0,0 +1,75 @@
+#include "global.h"
+#include "test/battle.h"
+
+// Please add Sandstorm interactions with move, item and ability effects on their respective files.
+SINGLE_BATTLE_TEST("Sandstorm deals 1/16 damage per turn")
+{
+    s16 sandstormDamage;
+
+    GIVEN {
+        PLAYER(SPECIES_SANDSLASH);
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        TURN {MOVE(player, MOVE_SANDSTORM);}
+    } SCENE {
+        MESSAGE("Foe Wobbuffet is buffeted by the sandstorm!");
+        HP_BAR(opponent, captureDamage: &sandstormDamage);
+   } THEN { EXPECT_EQ(sandstormDamage, opponent->maxHP / 16); }
+}
+
+SINGLE_BATTLE_TEST("Sandstorm multiplies the special defense of Rock-types by 1.5x", s16 damage)
+{
+    u16 move;
+    PARAMETRIZE{ move = MOVE_SANDSTORM; }
+    PARAMETRIZE{ move = MOVE_CELEBRATE; }
+    GIVEN {
+        PLAYER(SPECIES_WOBBUFFET) ;
+        OPPONENT(SPECIES_NOSEPASS);
+    } WHEN {
+        TURN { MOVE(opponent, move); }
+        TURN { MOVE(player, MOVE_SWIFT); }
+    } SCENE {
+        HP_BAR(opponent, captureDamage: &results[i].damage);
+    } FINALLY {
+        EXPECT_MUL_EQ(results[0].damage, Q_4_12(1.5), results[1].damage);
+    }
+}
+
+SINGLE_BATTLE_TEST("Sandstorm damage does not hurt Ground-type Pokémon")
```

Comment:
Combine these three into a single test using `PARAMETRIZE`

Reply:
How do I do this? All tests use different pokemon which display different messages.

Reply:
Alternatively, `"Clear Amulet prevents stat reducing effects"`